Loenn/lang/en_gb.lang

```diff
@@ -14,6 +14,12 @@ entities.CommunalHelper/DreamJellyfish.attributes.description.tutorial=Whether t
 # Dream Booster
 entities.CommunalHelper/DreamBoosterAny.placements.name.dream_booster=Dream Booster (Any Direction)
 
+#Dream Spring
+entities.CommunalHelper/DreamSpringUp.placements.name.up=Dream Spring (Up)
+entities.CommunalHelper/DreamSpringRight.placements.name.right=Dream Spring (Right)
+entities.CommunalHelper/DreamSpringLeft.placements.name.left=Dream Spring (Left)
+entities.CommunalHelper/DreamSpringDown.placements.name.down=Dream Spring (Down)
+
```

Comment:
???

Reply:
It was an idea I started developing but it wasn't supposed to ship in this PR, i'm removing it